tests/tasks/addons/cwagent.yaml

```diff
@@ -0,0 +1,59 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: eks-addon-cwagent
+  namespace: tekton-pipelines
+spec:
+  description: |
+    This Task can be used to create a EKS CW Agent Deamonset on EKS cluster
+  params:
+  - name: cluster-name
+    description: The name of the EKS cluster you want to spin managed nodegroups for.
```

Comment:
seems like a copy paste error in description

Reply:
:D yeah. Good catch! I will remove it.